artifacts.go

```diff
@@ -27,6 +26,9 @@ import (
 	"github.com/urfave/cli"
 )
 
+// VERSION of the mender-artifact CLI tool
+var VERSION = "0.1"
```

Comment:
Maybe we should consider using the [same mechanism that we do for Mender](https://github.com/mendersoftware/mender/blob/master/Makefile#L18-L21). Hardcoded strings have a tendency to go out of date. What do you think?